pkgs/applications/misc/ebook2cw/default.nix

```diff
@@ -0,0 +1,35 @@
+{ stdenv, fetchurl, lame, libvorbis }:
+
+stdenv.mkDerivation rec {
+
+  name = "ebook2cw-${version}";
+  version = "0.8.2";
+
+  src = fetchurl {
+    url = "http://fkurz.net/ham/ebook2cw/${name}.tar.gz";
+    sha256 = "0n2p1i8cnnl2y3cdw8knx1vvs39sid45fr59p6f4vrprmb37l9yq";
+  };
+
+# TODO: Use SVN once fetchsvn stops stripping the trailing slash,
+# otherwise we get a build failure due to a redirect cycle.
```

Comment:
Is there an issue open with this bug? I can't find one.

Reply:
I will add one.

Reply:
Created #31441

Reply:
Looks like it works if we change `http://` to `svn://`.  Updated to use `fetchsvn`.